### PR TITLE
openstack-k8s: add .prow-ci.env support

### PR DESCRIPTION
**Pull request by dprince**
This file can host custom ENV variables per project. Currently used only in the build commands

### CODE REVIEW

File: ci-operator/step-registry/openstack-k8s-operators/build/openstack-k8s-operators-build-commands.sh [hunk 1]
```diff
@@ -99,6 +99,12 @@ function build_push_operator_images {
 
   unset GOFLAGS
   pushd ${OP_DIR}
+
+  # custom per project ENV variables
+  if [ -f .prow_ci.env ]; then
```
Comment (viroel) [inline ci-operator/step-registry/openstack-k8s-operators/build/openstack-k8s-operators-build-commands.sh:104]: we could leverage this to all other steps, but it is ok for now just to validate the proposal.

**Comment by viroel** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-openstack-operator-main-openstack-operator-build-deploy-tempest

for sanity check

**Comment by viroel** [comment]
/lgtm

code looks good, thanks

**Comment by Sandeepyadav93** [comment]
/pj-rehearse ack

Looks good to me, I like the idea and this way we can override prow related vars.

**Pull request merged**